packages/react/src/BacktraceClient.ts

```diff
@@ -0,0 +1,38 @@
+import {
+    BacktraceAttributeProvider,
+    BacktraceCoreClient,
+    BacktraceRequestHandler,
+    BacktraceStackTraceConverter,
+} from '@backtrace/sdk-core';
+import { AGENT } from '@backtrace/browser';
+import { BacktraceConfiguration } from '@backtrace/browser';
+import { BacktraceClientBuilder } from '@backtrace/browser';
+
+declare global {
+    interface Window {
+        backtraceClient?: BacktraceClient;
+    }
+}
```

Comment:
Instead of making a global variable on `window`, maybe we could make `BacktraceClient` have a static `instance` variable?

Reply:
Sure thing. I'll push that up after lunch.

Reply:
Fixed, let me know what you think

Reply:
The instance can be `undefined` here, right? I'm not sure why Typescript doesn't raise an error here:
```
public static instance: BacktraceClient;
```

I was thinking, maybe change this to a getter?

```
private static _instance: BacktraceClient;

public static get instance(): BacktraceClient {
  if (!this._instance) {
    throw new Error('BacktraceClient is uninitialized. Call "BacktraceClient.initialize" function first.');
  }
}
```

That way, we have more robust error handling instead of cryptic `TypeError`s.

Reply:
I like it! Updated.